docs/using/backup.md

```diff
@@ -126,199 +105,117 @@
-            - *Start time, h/m*: The hour and minute for the backup.
-                - In the first field, select one or more hours (*00* to *23*, *00* is midnight).
-                - In the second field, select one or more minutes (*00* to *59*).
-
-        - *Retention*: How many backups to keep. For unlimited, use `0` (zero).
-        - *Retry mode*: In case of error, either let PMM retry the backup again ("Auto") or do it again yourself ("Manual").
-        - *Retry times and interval*: If "Auto" retry mode is selected, the maximum number of retries - up to 10 - and the interval between them - up to 8 hours - can be set here.
-        - *Enable*: Deselect to define the scheduled backup without enabling it.
-
-        ![!](../_images/PMM_Backup_Management_Schedule.png)
-
-        !!! note ""
-            For this release ({{release}}), times are UTC.
-
-1. Click *Schedule*.
-
-1. A new entry will appear in the list.
+- Check out the current [MongoDB supported configurations and limitations](mongodb_limitations.md).
+
+## [Make a backup](#make-a-backup)
+
+To create a backup:
+
+1. Go to  <i class="uil uil-history"></i> **Backup > All Backups**.
+2. Click <i class="uil uil-plus-square"></i> **Create Backup**.
+3. Specify the type of backup that you want to create: **On Demand** or **Schedule Backup**.
+4. Enter a unique name for this backup.
+5. Choose the service to back up from the Service name drop-down menu. This automatically populates the **DB Technology** field.
+6. Select whether you want to create a **Physical** or **Logical** backup of your data, depending on your use case and requirements. For MySQL, you only the **Physical** data model is available.
+7. Choose a storage location for the backup. MysQL currently only supports storing backups to Amazon S3. If no options are available here, see the [Create a storage location](#create-a-storage-location) section above.
+8. If you're creating a schedule backups, also specify the backup type, the schedule, and a retention policy for your backup:
+    - **Backup Type**: currently, PMM supports both **Full** and Point-in-type recovery **(PITR)** backup types for MongoDB. However, the PITR option is only available for the **Logical** data model. For MySQL, only the **Full** type is supported.
+    - **Shedule**: configure the frequency and the start time for this backup.  Make sure that the the schedule you specify here does not overlap with production hours, and doesn't create an overhead on the production environment.
+    - **Retention**: this option is only available for PITR backup types. If you wand to keep an unlimited number of backup artifacts, type `0`.
+9. Expand **Advanced Settings** to specify the settings for retrying the backup in case of any issues. You can either let PMM retry the backup again (**Auto**), or do it again yourself **Manual**. Auto retry mode enables you to select up to ten retries and an interval of up to eight hous between retries.
```

Comment:
```suggestion
9. Expand **Advanced Settings** to specify the settings for retrying the backup in case of any issues. You can either let PMM retry the backup again (**Auto**), or do it again yourself **Manual**. Auto retry mode enables you to select up to ten retries and an interval of up to eight hours between retries.
```